data/npc/lib/npcsystem/modules.lua

```diff
@@ -960,7 +960,7 @@ if not Modules then
 			return false
 		end
 
-		if shopItem.buy == -1 then
+		if shopItem.buy == 0 then
```

Comment:
I think it should be a valid value to buy/sell for free. To not me buyable/sellable the value in Lua table should be nil.

Reply:
I don't think the client can handle that (haven't tested), since cip sell items for 1 gp at testserver assistant and not 0 (for free)